colossalai/pipeline/schedule/interleaved_pp.py

```diff
@@ -18,7 +17,7 @@
 from .base import PipelineSchedule
 
 
-def _wait_p2p(wait_handles: List[torch.cuda.Event]) -> None:
+def _wait_p2p(wait_handles: List[get_accelerator().Event]) -> None:
```

Comment:
`get_accelerator().Event` is incomplete type during coding time

Reply:
updated.